src/css/ContactLinks.scss

```diff
@@ -1,29 +1,32 @@
 /*------------------------------------*\
```

Comment:
I don't think the comment at the top is necessary when in smaller component CSS files. But I like the emphasis on organization.